src/snowflake/snowpark/lineage.py

```diff
@@ -424,10 +424,13 @@ def _get_name_and_version(self, graph_entity: Dict[str, Any]):
         if user_domain in self._versioned_object_domains:
             if user_domain == _UserDomain.FEATURE_VIEW:
                 if "$" in name:
-                    parts = name.split("$")
+                    had_quotes = name.startswith('"') and name.endswith('"')
```

Comment:
nit: has_quotes

Reply:
oops, I didnt see this. I have another PR open, I will fix this in that one